internal/resolution/variable_sources/entity/bundle_entity.go

```diff
@@ -11,6 +11,14 @@ import (
 )
 
 const PropertyBundlePath = "olm.bundle.path"
+const PropertyBundleMediaType = "olm.bundle.mediatype"
+
+type MediaType string
+
+const (
+	MediaTypePlain    = "plain+v0"
+	MediaTypeRegistry = "registry+v1"
+)
```

Comment:
This is likely not the right place to define these in the long run, but probably fine for now. Add a TODO comment about finding a more permanent home?